aws-quickstart/quickstart-aws-industrial-machine-connectivity
# Unify Integration Update zip file

### Changes:
Update Unify lambda zip files.  

script  "unify_scripts/deploy_to_repo.sh"  build zip files from the source code and upload to test repo "s3://imc-unify-demo/"

### How to test 

1. run "unify_scripts/deploy_to_repo.sh" to build and deploy zip file to test repo "imc-unify-demo"
2. Create stacks from test repo
3. Make sure the lamdba functions works and zip code have the right version
